### PR TITLE
AuthPage Login Signup added

### PR DESCRIPTION
**Pull request by surbhi84**
Converted html to react for static login and signup components.

### CODE REVIEW

File: src/components/auth/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export function Signup() {
+  return (
+    <>
+      <main className="flex-center">
+        <div
+          className={`${auth["signup-card"]} ${auth["auth-card"]} flex-col flex-center gap-md`}
+        >
+          <h3 className={auth["auth-heading"]}>Sign up</h3>
+
+          <label className={`flex-col gap-sm ${auth["full-width"]}`}>
```
Comment (prakashsakari) [inline src/components/auth/Signup.jsx]: Just a suggestion - you can create a component for elements that follow the common structure like first name, last name, email id, and reuse it.

**Review by prakashsakari — APPROVED** [review]
1. UI looks neat and clean
2. You can align the password hide/show icon with Password Label, as it is not aligned

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 function App() {
+  const [isHome, setIsHome] = useState(true);
+
   return (
     <div className="App">
-      <Navbar />
+      <Routes>
+        <Route path="/" element={<Navbar isHome={true} />} />
+        <Route path="*" element={<Navbar isHome={false} />} />
+      </Routes>
```
Comment (rohanmathur91) [inline src/App.js:16]: Can you explain this?

File: src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -1,77 +1,104 @@
 import navbar from "./navbar.module.css";
 import { Carousel } from "../carousel/Carousel";
 import { Routes, Route, Link } from "react-router-dom";
+import { useState } from "react";
+import { useEffect } from "react";
```
Comment (rohanmathur91) [inline src/components/navbar/Navbar.jsx]: Put it in one import.

**Review by rohanmathur91 — APPROVED** [review]
- Nice use of CSS modules
 - Import and exports for components are clean
 - Over all code structure is good

File: src/App.js (hunk 2 above)
Comment (surbhi84) [reply src/App.js:16]: I have used different classes for my navbar for different page renders, one contains absolute positioning as i needed my navbar to layer over my carousel, which was not required in other pages.
the above code sends a prop as true or false, which will tell my navbar if it is getting rendered in home page or anywhere else.

Comment (rohanmathur91) [reply src/App.js:16]: So it is only home page where you want that style ?, if so then you can check the current path using the useLocation hook provided by react-router-dom and apply the class according to it

**Pull request merged**